File: WorldFeed/Client/src/app/core/Public/Models/Interfaces/DTO/IExtendedTweet.ts
import {ITweetEntities} from "../../Entities/ITweetEntities";

export interface IExtendedTweet {
  text: string;

  fullText: string;

  displayTextRange: number[];
  legacyEntities: ITweetEntities;
  extendedEntities: ITweetEntities;
}

File: Sharebook/Server/src/Services/Post/Sharebook.Post.API/DTO/IExtendedTweet.cs
namespace Sharebook.Post.DTO
{
    using Sharebook.Common.Public.Models.Entities;
    public interface IExtendedTweet
    {
        string Text { get; set; }

        string FullText { get; set; }

        int[] DisplayTextRange { get; set; }

        ITweetEntities LegacyEntities { get; set; }

        ITweetEntities ExtendedEntities { get; set; }
    }
}
